src/webhooks/webhook.ts

```diff
@@ -0,0 +1,61 @@
+import { runCommand } from '@lib/runCommand.js';
+import { linkMergeRequest, linkBranch, referenceToRecordFromTitle, linkPullRequestToRecord } from '../lib/fields.js';
+
+aha.on('webhook', async ({ headers, payload }) => {
+  const event = payload.eventType;
+
+  console.log(`Received webhook '${event}' ${payload.event_type || ''}`);
+
+  switch (event) {
+    case 'git.branch.created':
+      await handleCreateBranch(payload);
+      break;
+    case 'git.pullrequest.created':
+    case 'git.pullrequest.updated':
```

Comment:
I believe this needs to handle a `git.pullrequest.merged` event as well – I don't see the status changing in Aha! Develop after merging/closing a PR.

Reply:
@trydionel  I could not find the `git.pullrequest.merged` event. I got the merged status via `git.pullrequest.updated` event.
![image](https://user-images.githubusercontent.com/79676156/152605230-3faf8109-51f6-481f-a5dc-9d0106a03033.png)